src/d3a/d3a_core/util.py

```diff
@@ -527,6 +537,25 @@ def convert_area_throughput_kVA_to_kWh(transfer_capacity_kWA, slot_length):
         if transfer_capacity_kWA is not None else 0.
 
 
+def find_timestamp_of_same_weekday_and_time(indict, time_slot, ignore_not_found=False):
```

Comment:
```suggestion
def find_object_of_same_weekday_and_time(indict, time_slot, ignore_not_found=False):
```
Initially I thought this method would return a `timestamp` but later looking into the implementation, realised it either is returning a value or profile having an offset of weekday_and_time